docker-compose.dev.yml

```
@@ -0,0 +1,90 @@
+services:
+  db:
+    image: "postgres"
+    restart: always
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
+    expose:
+      - 5432
+    volumes:
+      - database-volume:/var/lib/postgresql
+    networks:
+      - slimevr-network
+  api:
+    build:
+      context: .
+    ports:
+      - 3000:3000
```

by default only expose the port to localhost
```suggestion
      - 127.0.0.1:3000:3000
```